lib/server.js

```diff
@@ -106,6 +106,8 @@ var GV_ARGS_SCHEMA = [
     { name: 'pnode', type: 'string' }
 ];
 
+var GDP_ARGS_SCHEMA = [];
```

Comment:
Should we fill this schema out? Or is this blank because the RPC doesn't require any args?

Reply:
Judging by the implementation (I didn't write this part of the code) it looks to be blank because the RPC doesn't require any arguments.  I can annotate that in a comment if we want.

Reply:
Right, I don't know if we actually need the schema in this case, but I just went ahead an created it since every other function uses one.

Reply:
Agreed, seems fine to keep as is. I think a comment would be good for this, because I'm not sure if there's anywhere else that describes that the function takes no args.

Reply:
:+1: I'll add that.